src/main/kotlin/com/github/patou/gitmoji/GitmojiLocale.kt

```diff
@@ -28,7 +28,7 @@ class GitmojiLocale {
         }
         val client = OkHttpClient().newBuilder().addInterceptor(SafeGuardInterceptor()).build()
         val request: Request = Request.Builder()
-            .url("https://raw.githubusercontent.com/caiyucong/gitmoji-intellij-plugin-chinese/master/src/main/resources/gitmojis-${defaultLanguage}.yaml")
+            .url("https://raw.githubusercontent.com/caiyucong/gitmoji-intellij-plugin-chinese/t/gitmojis-${defaultLanguage}.yaml")
```

Comment:
Perhaps I could use a file to record the locale codes that have already been implemented, separated by commas. This would make maintenance simpler.

I would just need to add the locale codes I've already implemented to the file. Although this goes against the idea of adding only one file, my method would be simpler.

Additionally, I wouldn't need to update and iterate through the code; I would only need to update my "t" branch.

If someone needs to add a translation, they could do so via a pull request (PR). If I were to use properties (languageCode, languageName, URL of the file...) for maintenance, I would still need to operate via PRs.

Simplifying our operations would make maintenance easier.

What do you think of my idea?

Reply:
You will not maintain all translation in your repos, if this it's better to add files in this repos and not in a custom repos.

For add a new language, with the current code, it's must add a new items in the list. And it's important to keep a local copy if the user don't have internet (or the internet is blocked from proxy for example).

Reply:
We can first access online resources. If the access fails, we will then access local resources.

If possible, we can place the translated files into your code repository. Prioritizing access to remote resources in this way would allow for better maintenance, avoiding the need to re-release the plugin. 

Additionally, we can add translation files by creating (gitmoji-{localeCode}.yaml).